app/Mcp/Tools/GetDayContextTool.php

```
@@ -152,7 +152,7 @@ protected function getTimeframeKey(string $input, Carbon $targetDate): string
 
         // For specific dates, determine relative position to today
         $today = Carbon::today();
-        $diffDays = $today->diffInDays($targetDate, signed: false);
+        $diffDays = $today->diffInDays($targetDate, absolute: false);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$diffDays'. (undefined)

(UnusedLocalVariable)